### ISSUE TITLE
Change Cover	Photo for Users by Administrator

### ISSUE THREAD

**Issue by CimborNet** [PHPfox-Official/phpfox-v4-feature-requests#370]
Why does not the administrator of the network have the ability to edit, change the position or delete the user-created cover photo for his profile?

I do not have anywhere such an option that would enable / disable it. If it is, then why is it not enabled by default for root?

It often happens that a new user can not or does not understand how to install/moving posions a cover photo on the his profile and asks to do it for him.

But even I, having the rights of an administrator on the site, I can not edit his profile cover.

Even when editing users on AdminCP, there is an option to change their avatar (Profile picture) but there is no way to change or delete their profile cover photo.

Why?

Users profiles.....
![9eebcb9e-42d5-11e7-9432-07ee9052907b](https://cloud.githubusercontent.com/assets/11414490/26557788/3f5153e0-44a4-11e7-9d75-fe809b208a4e.jpg)
![a1ebf2b0-42d5-11e7-9b7a-f6b0db03b3c4](https://cloud.githubusercontent.com/assets/11414490/26557790/45232b5e-44a4-11e7-9506-cd1bd20f6d94.jpg)
![2ce1c642-42d6-11e7-8f0f-72cd6ec59152](https://cloud.githubusercontent.com/assets/11414490/26557793/495280da-44a4-11e7-8287-488c1fdd3604.jpg)

For example, the administrator has the ability to edit or change the cover for groups and pages. But for the user profile there is no such possibility.

![0ae90b4e-42d7-11e7-9026-10b9dd72c8cc](https://cloud.githubusercontent.com/assets/11414490/26557782/35dfda2a-44a4-11e7-907f-776f764dcfe9.jpg)

**Comment by Erick000** [comment]
This is true, we do need to be able to add/delete/modify users cover photo IF any...

**Comment by neil96** [comment]
We accepted feature: admin can delete cover photo of all user.

**Comment by spirogg** [comment]
@CimborNet @harrison05 I know it shows in 4.7.9 that they added this feature, but if you went to adminCP >Members>User Group Settings> 
under apps select user
under Groups select administrator
then at bottom turn on feature: Can members of this user group log in as another user without entering a password?

no you can search users login as that user and do whatever you want 

hope this also helps :)
Spiro
